signal_scanner_bot/env.py

```diff
@@ -116,6 +124,25 @@ def _cast_to_path(to_cast: str) -> Path:
     return Path(to_cast)
 
 
+def _format_hashtags(to_cast: str) -> List[str]:
+    hashtags = _cast_to_list(to_cast)
+    output_hashtags = list()
```

Comment:
For `list` and `dict`, the literal instantiations are actually faster (and more common)
```suggestion
    output_hashtags = []
```

Reply:
Huh, that's what I get for believing everything I read on the internet I guess.